<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>464</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="filterLayout">
    <item>
     <widget class="QLineEdit" name="filterLineEdit">
      <property name="placeholderText">
       <string>Filter...</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="filterColumnLabel">
      <property name="text">
       <string>Column:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="filterComboBox">
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filterMatchCase">
      <property name="text">
       <string>Match Case</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filterWholeWord">
      <property name="text">
       <string>Whole Word</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkAll">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>464</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Check All</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>500</width>
     <height>250</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="headerVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="headerDefaultSectionSize">
    <number>74</number>
   </attribute>
   <attribute name="headerShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="headerStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
